src/backend/distributed/commands/alter_table.c

```diff
@@ -1580,7 +1571,8 @@ ReplaceTable(Oid sourceId, Oid targetId, List *justBeforeDropCommands,
 			 */
 			List *nonStoredColumnNameList = GetNonGeneratedStoredColumnNameList(sourceId);
 			char *insertColumnString = StringJoin(nonStoredColumnNameList, ',');
-			appendStringInfo(query, "INSERT INTO %s (%s) SELECT %s FROM %s",
+			appendStringInfo(query,
+							 "INSERT INTO %s (%s) OVERRIDING SYSTEM VALUE SELECT %s FROM %s",
```

Comment:
Do you know if `OVERRIDING SYSTEM VALUE` adds a significant overhead in performance? If that is the case, we may want to include an `if` statement here and use `OVERRIDING SYSTEM VALUE` only if needed.

Reply:
I couldn't find any evidence for that statement to introduce performance overhead.